tests/driver_at86rf215/main.c

```diff
@@ -74,8 +77,125 @@ static int cmd_enable_batmon(int argc, char **argv)
     return res;
 }
 
+static int cmd_set_trim(int argc, char **argv)
+{
+    if (argc < 2) {
+        printf("usage: %s <trim>\n", argv[0]);
+        return 1;
+    }
+
+    uint8_t trim = atoi(argv[1]);
+
+    if (trim > 0xF) {
+        puts("Trim value out of range");
+        return 1;
+    }
+
+    gnrc_netif_t *netif = gnrc_netif_get_by_type(NETDEV_AT86RF215, 0);
+
+    if (netif == NULL) {
+        puts("No at86rf215 radio found");
+        return 1;
+    }
+
+    at86rf215_t* dev = (at86rf215_t*)netif->dev;
+
+    printf("setting trim to %u fF\n", 300U * trim);
+    at86rf215_set_trim(dev, trim);
+
+    return 0;
+}
+
+static int cmd_set_clock_out(int argc, char **argv)
+{
+    const char *keys[] = {
+        [AT86RF215_CLKO_OFF]    = "off",
+        [AT86RF215_CLKO_26_MHz] = "26",
+        [AT86RF215_CLKO_32_MHz] = "32",
+        [AT86RF215_CLKO_16_MHz] = "16",
+        [AT86RF215_CLKO_8_MHz]  = "8",
+        [AT86RF215_CLKO_4_MHz]  = "4",
+        [AT86RF215_CLKO_2_MHz]  = "2",
+        [AT86RF215_CLKO_1_MHz]  = "1",
+    };
```

Comment:
Why not using `atoi()`? It is pulled in anyway. `atoi("off")` will return `0`, btw. So one could use both `"off"` and `"0"` to disable clock output.

Reply:
what if someone sets it to 6? Silently to the next value?

Reply:
How about?
```C
static inline bool is_freq_valid(int f_mhz)
{
    /* Allowed are powers of two up to 32, and 26 */
    if ((x > 32) || (!x)) {
        return false;
    }
    if (x == 26) {
        return true;
    }
    if (!((x - 1) & x)) {
        /* power of two */
        return true;
    }
    return false;
}
```

Or?
```C
static inline bool is_freq_valid(unsigned f_mhz)
{
    const uint32_t mask = BIT0 | BIT1 | BIT3 | BIT7 | BIT15 | BIT25 | BIT31;
    f_mhz--;
    if (f_mhz > 32)) {
        return false;
    }

    return (mask & (1 << f_mhz));
}
```

Reply:
Do you really think that's better?
This is test code, there is no need to squeeze out the last byte - I think readability is more important here.